shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java

```diff
@@ -492,6 +492,12 @@ public void onTrimMemory(int level) {
     flutterFragment.onTrimMemory(level);
   }
 
+  @Override
+  protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+    super.onActivityResult(requestCode, resultCode, data);
```

Comment:
We are forwarding onRequestPermissionResult only to the fragment. Should we also call super? or should we remove super from here?

Reply:
Since the request went out through the Activity, we probably want to call `super` to allow Android to do any internal accounting that it might want to do. It's actually our forwarding of the call to the `Fragment` that is unusual.